drivers/python/rethinkdb/net.py

```diff
@@ -346,14 +368,27 @@ def sendall(self, data):
                 self.close()
                 raise
 
+    def _get_ssl_context(self, ca_certs):
```

Comment:
For the purpose of sharing code with `net_tornado.py`, it may be useful to pull this out of `SocketWrapper`.  It doesn't appear to use any member data.